builder/bflag.go

```diff
@@ -12,28 +13,32 @@ const (
 	stringType
 )
 
-type BuilderFlags struct {
+// BFlags contains all flags information for the builder
+type BFlags struct {
```

Comment:
Since we have bflag.go, I thought we already consider bflag for short of buildflag, and bflag would be more meaningful than just flag. Personally I don't like that common word as a variable or method's name, it's painful when we jump between these tags :disappointed: 

If you insist, I surely can change the name :smile: